spec/content/automate/ManageIQ/Infrastructure/VM/Provisioning/Placement.class/__methods__/vmware_best_fit_least_utilized_spec.rb

```diff
@@ -20,6 +20,16 @@
   let(:root_object) { Spec::Support::MiqAeMockObject.new(:miq_provision => svc_miq_provision) }
   let(:ae_service)  { Spec::Support::MiqAeMockService.new(root_object) }
 
+  it 'requires miq_provision attribute in root object' do
+    new_service = Spec::Support::MiqAeMockService.new(Spec::Support::MiqAeMockObject.new)
+    expect { described_class.new(new_service).main }.to raise_error(RuntimeError, "ERROR - miq_provision object not passed in")
```

Comment:
@lfu does it make sense that the message be checked using a regular expression
e.g.
```
expect { raise "oops" }.to raise_error(RuntimeError, /op/)
```
https://relishapp.com/rspec/rspec-expectations/v/2-11/docs/built-in-matchers/raise-error-matcher

This way the spec wont fail if someone where to make a slight change to the message

Reply:
@lfu 
Should the 2 error messages be consistent
"ERROR - miq_provision object not passed in"
"ERROR - VM not specified"



also is ERROR needed since this is going to logged as an error anyway in 

https://github.com/ManageIQ/manageiq-content/blob/a50ea6c95dfb2e2b51a69720e67f0fae973d1733/content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb#L57